tests/httpClient.test.js

```diff
@@ -2078,6 +2080,56 @@ tap.test('httpClient', test => {
 
       myServiceNameScope.done()
     })
+
+    innerTest.test('log correctly - redact data', async assert => {
+      const data = []
+      const logStream = new PassThrough()
+        .on('data', (streamData) => {
+          data.push(Buffer.from(streamData, 'utf8').toString())
+        })
+
+      const myServiceNameScope = nock(MY_AWESOME_SERVICE_PROXY_HTTP_URL)
+        .replyContentLength()
+        .post('/foo')
+        .delay(101)
```

Comment:
Why this `.delay`? Not advisable in tests unless strictly needed

Reply:
I started by copying a text above. Its usefulness for this test is low

Reply:
```suggestion
```